libs/unity/library/Runtime/Scripts/Media/MicrophoneSource.cs

```diff
@@ -61,17 +61,25 @@ protected override async void OnEnable()
         }
 #endif
 
-        protected override async Task CreateAudioTrackSourceAsyncImpl()
+        protected virtual async Task OnEnable()
```

Comment:
Why virtual?

Reply:
Why indeed ¯\_(ツ)_/¯
I think that might have been an automated refactor removing the implementation in the base class, because I don't see why I'd have added that in a leaf class.
Also I just noted there are 2 implementations of `OnEnable()` now on UWP, will merge.

Reply:
Fixed.